2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,87 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2020-13671",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
-        ]
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "CVE_data_meta": {
+    "ID": "CVE-2020-13671",
+    "ASSIGNER": "cve@mitre.org",
+    "STATE": "PUBLIC"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Drupal ",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Drupal Core",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "9.0",
+                      "version_affected": "<",
+                      "version_value": "9.0.8",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "8.9",
+                      "version_affected": "<",
+                      "version_value": "8.9.9",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "8.8",
+                      "version_affected": "<",
+                      "version_value": "8.8.11",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "7",
+                      "version_affected": "<",
+                      "version_value": "7.74",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
     }
-}
\ No newline at end of file
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "Remote code execution"
+          }
+        ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Remote code execution vulnerability in Drupal Core allows an attacker to upload files leading to files being interpreted as the incorrect extension and served as the wrong MIME type or executed as PHP for certain hosting configurations."
+      },
+      {
+        "lang": "eng",
```

Comment:
The system doesn't know how to handle two descriptions for the same language.  You will need to consolidate the descriptions.